packages/client/src/pages/Leaderboard/Leaderboard.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+import { breakpoints } from 'components/App/constants';
+import { Label } from 'src/design/Label';
+import LeaderboardRow from 'components/LeaderboardRow';
+import { leaderboardConsts } from 'components/LeaderboardRow/consts/leaderboardConsts';
+
+const { mobileM } = breakpoints;
+
+const Leaderboard = () => {
+  return (
+    <Wrapper>
+      <LeaderboardComponent>
+        <LeaderboardLabel>Leaderboard</LeaderboardLabel>
+        <Column>
+          {leaderboardConsts.map(({ rank, nickname, score, isMine }) => (
+            <LeaderboardRow key={rank} rank={rank} nickname={nickname} score={score} isMine={isMine} />
+          ))}
```

Comment:
Тут можно не деструктуризировать, а просто сделать так например:

```suggestion
          {leaderboardConsts.map((props) => (
            <LeaderboardRow key={props.rank} {...props} />
          ))}
```

Reply:
Хороший способ сократить код! 
Но тут же возникает вопрос читабельности, насколько это удобнее для дальнейшего использования такого кода?

Reply:
>Но тут же возникает вопрос читабельности, насколько это удобнее для дальнейшего использования такого кода?

Согласен, когда нужно будет дебажить такого вида компонент, то придется вникать в то, что передается
Но в целом из-за типизации пропсов у нас читабельность сохраняется и все норм